src/mock-data/quote.ts

```diff
@@ -4,56 +4,69 @@ import {
   QuoteRequestBody,
 } from '@fiatconnect/fiatconnect-types'
 
-const quoteBrazilCREAL: Omit<QuoteRequestBody, 'address'> = {
+// NOTE: quotes here must include both amounts, tests will omit one of the
+// fields and test whether the endpoint supports fetching quotes for both
+// amounts
+type MockQuote = Omit<QuoteRequestBody, 'address'> &
+  Required<Pick<QuoteRequestBody, 'fiatAmount' | 'cryptoAmount'>>
+
+const quoteBrazilCREAL: MockQuote = {
   cryptoType: CryptoType.cREAL,
   fiatType: FiatType.BRL,
   cryptoAmount: '10',
+  fiatAmount: '10',
   country: 'BR',
 }
 
-const quoteOutNigeriaCUSD: Omit<QuoteRequestBody, 'address'> = {
+const quoteOutNigeriaCUSD: MockQuote = {
   cryptoType: CryptoType.cUSD,
   fiatType: FiatType.NGN,
   cryptoAmount: '10',
+  fiatAmount: '7560',
```

Comment:
for some providers, 10 is at or near the minimum quote offered. Given that Naira can fluctuate in value, we might want to increase cryptoAmount too so there's more of a buffer. Otherwise if Naira decreased in value, this could drop below the minimum quote amount offered by the provider, assuming that minimum amount is denominated in dollars

Reply:
same with the other mock quotes for 10 cUSD that now have a fiat amount

Reply:
I updated all quotes to be 20 cUSD, and also dropped the in / out in the field names since some objects were just duplicates